1-sum-app/sum.html

```diff
@@ -0,0 +1,14 @@
+<html>
+    <head>
+        <title>1-sum-app</title>
+    </head>
+        <body>
+           <div class="wrapper-content">
+            <br>
```

Comment:
Please avid using <br> in html. If we need to move the element to next line its better to wrap it in `<div> </div>`